src/Lachain.Core/Config/ConfigManager.cs

```diff
@@ -437,6 +441,29 @@ private void _UpdateConfigToV15()
             _SaveCurrentConfig();
         }
 
+        // version 16 of config should contain updated blockchain.TargetBlockTime (4000 ms) 
+        // and new blockchain.TargetTransactionsPerBlock (800) 
+        private void _UpdateConfigToV16()
+        {   
+            const int newTargetBlockTime = 4000; //ms
+            const int newTargetTransactionsPerBlock = 800; 
+
+            var blockchain = GetConfig<BlockchainConfig>("blockchain") ??
+                            throw new ApplicationException("No blockchain section in config");
+
+            blockchain.TargetBlockTime = newTargetBlockTime;
```

Comment:
Let's replace this value only if it was 5000 before

Reply:
can it be anything other than 5000 before? if so, when? and why should we be not changing it in that case?

Reply:
it could be different from 5000 if someone has changed it manually in ts own config. In this case let him has its custom value.